ci-operator/config/openshift/eventrouter/openshift-eventrouter-master.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.15
+    tag: golang-1.17
```

Comment:
I think we should go to go1.18 as that is the latest version.

Reply:
@Red-GV updated.  please re-review